src/commands/spork.ts

```diff
@@ -0,0 +1,76 @@
+import exitCodes from '../lib/exit-codes'
```

Comment:
Don't bother reviewing this file.

Reply:
![Don't tell me what I can't do](https://i.giphy.com/media/DDTG2I2R4hDUY/giphy.webp)